README.md

```diff
@@ -0,0 +1,41 @@
+
+<section>
+</div>
```

Comment:
Why is there a closing < / div > there?

Reply:
Tested with a different workflow badge but didnt seem relevant to this project. Forgot to delete!